scrapscript_tests.py

```diff
@@ -4051,5 +4052,51 @@ def test_pretty_print_variant(self) -> None:
         self.assertEqual(pretty(obj), "#x (a -> b)")
 
 
+class ServerCommandTests(unittest.TestCase):
+    def setUp(self) -> None:
+        import threading
+        import time
+        import os
+        import socket
+        import argparse
+        from scrapscript import server_command
+
+        # Find a random available port
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            s.bind(("127.0.0.1", 0))
+            self.host, self.port = s.getsockname()
+
+        args = argparse.Namespace(
```

Comment:
Using argparse in a test indicates to me that there is an impedance mismatch in the API design. It indicates that there should be a differently-testable interface that doesn't rely on CLI arguments. Probably fine for now but in the future we'll likely want a `serve_yard` function with only positional parameters or something